.github/workflows/tests.yml

```diff
@@ -225,8 +225,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: update-test (Ubuntu 22.04)
-            runs-on: ubuntu-22.04
+          - name: update-test (Ubuntu Latest)
+            runs-on: ubuntu-latest
```

Comment:
May want to keep this one? Not sure.

Reply:
This one runs tests to check if `brew update` still works. I think it's not a big deal to let it run on latest.
If it was OS-critical we would test on ubuntu versions we support.

Reply:
Agreed with @iMichka.